ElDorito/Source/Modules/ModuleGame.cpp

```diff
@@ -1258,4 +1318,49 @@ namespace Modules
 			std::sort(MapList.begin(), MapList.end());
 		}
 	}
+
+	void ModuleGame::onVKeyboardInput(std::wstring input)
+	{
+		if (!variant_vector.size())
+			return;
+		input.substr(0, 15);
+		if (input != mapVariant_name) {
+			const char* new_name = reinterpret_cast<const char*>(input.c_str());
+
+			// Offset for current variant name - original name will remain untouched
+			int string_offset_1 = 0x48;
+
+			for (int i = 0; i <= 31; i++) {
```

Comment:
once again can't you convert the string to wstring using `std::codecvt` and write the string using `strcpy_s`?